interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/TyperSpec.scala

```diff
@@ -104,6 +104,12 @@ class TyperSpec extends AnyFunSuite with Matchers with ValidatedValuesDetailedMe
       Typed.record(Map("nestedRecordKey" -> TypedObjectWithValue(Typed.typedClass[Int], 2)))
   }
 
+  test("validate properly maps with maps as keys") {
```

Comment:
are we able to provide an integration test (based on scenario builder) where we are able to show an example of how it relates to scenario definition?